doc/suiterc.tex

```diff
@@ -1372,6 +1418,129 @@ \subsection{[runtime]}
 \item {\em default:} False
 \end{myitemize}
 
+\paragraph[{[[[}events{]]]}]{[runtime] $\rightarrow$ [[\_\_NAME\_\_]] $\rightarrow$ [[[events]]]}
+\label{TaskEventHandling2}
+
+Cylc can call nominated event handlers when certain task events occur. This
+section configures general and built-in task event handlers;
+see~\ref{SuiteEventHandling} for suite event hooks. See
+~\ref{TaskEventHandling} for specific task event handlers.
+
+Event handler commands can be located in the suite \lstinline=bin/= directory,
+otherwise it is up to you to ensure their location is in \lstinline=$PATH= (in
+the shell in which cylc runs, on the suite host). The commands should require
+very little resource to run and should return quickly.
+
+Each task event handler can be specified as a list of shell scripts or a shell
+script templates.
+
+A shell script template may have any or all of these patterns which will be
+substituted with actual values:
+\begin{myitemize}
+    \item %(event)s: event name
+    \item %(suite)s: suite name
+    \item %(point)s: cycle point
+    \item %(name)s: task name
+    \item %(submit\_num)s: submit number
+    \item %(id)s: task ID (i.e. %(name)s.%(point)s)
+    \item %(message)s: event message, if any
+\end{myitemize}
+
+Otherwise, the shell script will be called with the following command line
+arguments:
+\begin{lstlisting}
+<task-event-handler> %(event)s %(suite)s %(id)s %(message)s
+\end{lstlisting}
+
+\subparagraph[handler]{[runtime] $\rightarrow$ [[\_\_NAME\_\_]] $\rightarrow$ [[[events]]] $\rightarrow$ handler}
+
+Specify a list of script or script templates as task event handlers.
+
+Item details:
```

Comment:
We haven't used "Item details" elsewhere (nothing wrong with it, but inconsistent).